contracts/DelFiPrice.sol

```diff
@@ -105,6 +176,24 @@ contract DelFiPrice is OpenOracleView {
         }
     }
 
+
+    /**
+     * @notice Returns the cToken address for symbol
+     * @param symbol The symbol to map to cToken address
+     * @return cToken The cToken address for the given symbol
+     */
+    function getCTokenAddress(string memory symbol) public view returns (address cToken) {
+        bytes32 symbolHash = keccak256(abi.encodePacked(symbol));
+        if (symbolHash == symbolEth) return cEthAddress;
```

Comment:
boy, I miss case/pattern match statements!

Reply:
yes, we can easily add it here.

Reply:
Actually no, it can not be added, not in solidity, in Yul.

Reply:
Yeah, to put this another way: Solidity is not about cool. Solidity is about unlearning every programming technique you've learned in the last 10 years and writing things out like you were teaching a toddler. Make it direct and simple.